pkg/configuration/config_manager/builder.go

```diff
@@ -349,39 +352,62 @@ func buildLazyRenderedConfigVolume(cmName string, manager *CfgManagerBuildParams
 
 func checkOrCreateConfigMap(referenceCM client.ObjectKey, scriptCMKey client.ObjectKey, cli client.Client, ctx context.Context, cluster *appsv1alpha1.Cluster, fn func(cm *corev1.ConfigMap) error) error {
```

Comment:
Seems changes to 'checkOrCreateConfigMap' has no relationship with "remove dependence on characterType", right?

Reply:
This is a bug. It supports create but cannot update. If the reloadAction script is adjusted, it will not take effect.

Reply:
I see, so it's a bug fix that is not related with "remove dependence on characterType" ?